env/posix/ocf_env.c

```diff
@@ -6,7 +6,11 @@
 #include "ocf_env.h"
 #include <sched.h>
 #include <execinfo.h>
+#include <sys/types.h>
+#include <setjmp.h>
+#include <cmocka.h>
```

Comment:
We shouldn't include cmocka headers in posix env files. Not sure about the setjmp.h - if this is specific to test framework, it shouldn't be here either.

Reply:
Removed and added during UT compilation.